docs/quickstarts/react.mdx

```diff
@@ -3,34 +3,85 @@ title: Use Clerk with React
 description: Learn how to use Clerk to quickly and easily add secure authentication and user management to your React application.
 ---
 
-import {Steps} from "nextra-theme-docs";
-import { Callout } from "nextra-theme-docs";
-import { YouTube } from "@/components/Youtube";
-import { Images } from "@/components/Images";
-import {Cards} from "@/components/Cards";
-import { CodeBlockTabs } from "@/components/CodeBlockTabs";
+<TutorialHero 
+  quickstart="react"
+  exampleRepo={[ 
+    {
+      title: "React Quickstart Repo",
+      link: "https://github.com/clerk/clerk-react-quickstart"
+      
+    }
+  ]}
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk"
+    },
+    {
+      title: "Create a React application using Vite",
+      link: "https://vitejs.dev/guide/#scaffolding-your-first-vite-project"
+    }
+  ]}
+>
+- Use Clerk with React
+- Install `@clerk/clerk-react`
+- Set environment keys
+- Wrap your app in `<ClerkProvider />`
+- Embed the `<UserButton />` and `<SignInButton />`
+</TutorialHero>
+
+
 
 # Use Clerk with React
 
 Learn how to use Clerk to quickly and easily add secure authentication and user management to your React application.
 
 <Steps>
 
+### Install React using Vite
```

Comment:
```suggestion
### Set up a React application using Vite
```

Reply:
If we have this section on how to create a react app with vite, then it probably shouldn't be in the "prerequisites", because it's actually a step in the tutorial - not a step before.